packages/push/src/push_vm/stack.rs

```diff
@@ -83,14 +84,16 @@ pub trait HasStack<T> {
     }
 }
 
-#[derive(thiserror::Error, Debug, Eq, PartialEq)]
+#[derive(thiserror::Error, Debug, Eq, PartialEq, Diagnostic)]
 pub enum StackError {
     #[error("Requested {num_requested} elements from stack with {num_present} elements.")]
+    #[diagnostic(severity(Warning))]
     Underflow {
         num_requested: usize,
         num_present: usize,
     },
     #[error("Pushed onto full stack of type {stack_type}.")]
+    #[diagnostic(help = "You might want to increase your stack size", severity(Warning))]
```

Comment:
Similar to the comment above, this is problem a symptom of a runaway program of some sort, and it's unlikely that (a) the user will ever see this and (b) that increasing the stack size will really help.